src/containers/UsernameTranslation.js

```diff
@@ -0,0 +1,163 @@
+import React, { useState, useEffect, useRef } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Link } from "react-router-dom";
+
+/*
+At the moment works ONLY with apiPath='seacat_auth'
+- userId (required, array) - user ids to be translated
+- apiPath - default (and only working so far) 'seacat_auth'
+
+Usage:
+
+import { UsernameTranslation } from 'asab-webui';
+...
+<UsernameTranslation app={props.app} id={[user1-id, user2-id, user3-id]} apiPath='seacat_auth' />
+
+*/
+
+export function UsernameTranslation(props) {
+
+	const id = Array.isArray(props.userId) ? props.userId : [props.userId] ;
+	const apiPath = props.apiPath ?? 'seacat_auth';
+	const { t } = useTranslation();
+	let API = props.app.axiosCreate(apiPath);
+	const [usernames, setUsernames] = useState({});
+	const USERNAMES_CACHE = "usernames";
+	const twoWeeks = 1000 * 60 * 60 * 24 * 14;
+	// test ids
+	// const kryton = 'mongodb:ext:618ced0fe6f6e38ab6a9c8b8';
+	// const rimmer = 'mongodb:ext:618be17866f6ee5de4ae5328';
+	// const id = [kryton,  rimmer];
+
+	const retrieveUserNames = async () => {
+		try {
+			let response = await API.put(`usernames`, id);
+			if (response.data.result !== "OK") {
+				throw new Error(t("TenantDetailContainer|Something went wrong, failed to fetch assigned credentials"));
```

Comment:
@petrKavulok This will definitely not be a `TenantDetailContainer` but something like `ASABUsernameTranslation`